File: AnimatedRating/app/src/main/java/com/rybarstudios/animatedrating/MainActivity.java
package com.rybarstudios.animatedrating;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CustomRating customRating = findViewById(R.id.custom_rating);
        customRating.setMaxRating(5);
        customRating.setDefaultRating(1);
    }
}

File: app/src/main/java/com/example/customratingbar/MainActivity.kt
package com.example.customratingbar

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.transition.Explode
import android.widget.TextView
import androidx.fragment.app.DialogFragment
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.fragment_rating.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btn_add_rating.setOnClickListener {
            val ratingFragment:DialogFragment = RatingFragment()
             ratingFragment.show(supportFragmentManager, "Rating Fragment")

        }
    }
    fun addView(){
        var textView= TextView(this)
        textView.text="${RatingBarView.name} ${RatingBarView.rating}"
        textView.textSize=20f
        layout_ratings.addView(textView)

    }

}
